extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcRecorder.java

```diff
@@ -246,6 +243,18 @@ public OidcProvider apply(JsonWebKeyCache jwks) {
                 });
     }
 
+    protected static Uni<JsonWebKeyCache> getJsonWebCacheUni(OidcProviderClient client, OidcTenantConfig oidcConfig) {
```

Comment:
Perhaps `getJsonWebKeySetUni`?

Reply:
@pedroigor hey Pedro, yes, I renamed it this way and also an internal  `JsonWebKeyCache` to `JsonWebKeySet`; when I named it `JsonWebKeyCache` I meant to convey it was a local JWK set cache but it is a JWK set after all :-), internally it used Jose4j `JsonWebKeySet` as well